gulpfile.js

```diff
@@ -171,6 +184,13 @@ gulp.task('compile', gulp.series([
 ]));
 
 /**
+ * Run tests against non-webpack assets
+ */
+gulp.task('test', gulp.parallel([
+  'test:accessibility',
```

Comment:
our test task should probably run the full lint too (ie including config files). unless testing is different from validation in this case

Reply:
A couple of things here:

There's a hard line between what Gulp does and what WebPack does. Basically, Gulp handles all assets that we can't get into WebPack.

1. We do linting on asset code via webpack, and have an npm script for everything else
1. This accessibility testing touches non-webpack assets, thus gets a Gulp task. Though there is an npm script for single file testing

Basically, I don't want to build out a gulp task for linting we already have done via WebPack.